src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -1262,8 +1322,40 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 					{
 						h.append( m_stepSize );
 						h.append( m_volumePadding );
+
+						if( m_volumeGrids )
+						{
+							m_volumeGrids->hash( h );
+						}
+
+						if( m_velocityGrids )
+						{
+							m_velocityGrids->hash( h );
+						}
+
+						h.append( m_velocityScale );
+						h.append( m_velocityFPS );
+						h.append( m_velocityOutlierThreshold );
 					}
 					break;
+				case GafferVDB::VDBObjectTypeId :
+					h.append( m_stepSize );
+					h.append( m_volumePadding );
+
+					if( m_volumeGrids )
```

Comment:
This is duplicating the code from the case above. I'll make a suggestions to improve this and some other probems in my main review.